documentation/asciidoc/topics/con_fips_mode_cr.adoc

```diff
@@ -0,0 +1,22 @@
+[id="fips-mode-CR_{context}"]
+= FIPS mode for your `Infinispan` CR
+
+[role="_abstract"]
+The {openshiftplatform} can use certain Federal Information Processing Standards (FIPS) components that ensure {openshiftshort} clusters meet the requirements of a FIPS compliance audit.
+
+If you enabled FIPS mode on your {openshiftshort} cluster then the {ispn_operator} automatically enables FIPS mode for your `{brandname}` CR.
+
+[IMPORTANT]
+====
+The client-certificate implementation does not support FIPS mode.
+For a FIPS-mode enabled `{brandname}` CR, you cannot configure the client-certificate implementation to secure an application that runs inside your `{brandname}` cluster.
+You can resolve this issue by setting `extraJvmOpts: "-Dcom.redhat.fips=false"` in the `spec.container` field in your `Infinispan` CR.
```

Comment:
This won't work, as the Infinispan webhook will prevent the Infinispan CR being created if Client cert is configured.

How about:

```
[IMPORTANT]
====
Client certificate authentication is not currently supported with FIPS mode. Attempts to create an Infinispan CR with `spec.security.endpointEncryption.clientCert` set to a value other than `None` will fail.
====
```

Reply:
Yes, this works. I updated the note as per your suggestion. 

Are you OK to approve the changes or do you see other changes?